pywemo/ouimeaux_device/__init__.py

```diff
@@ -208,6 +198,8 @@ def subscription_update(self, _type, _params):
             try:
                 self._state = int(self.parse_basic_state(_params).get("state"))
             except ValueError:
+                LOG.warning("Unexpected BinaryState value `%s` for device %s. "
```

Comment:
Lets log an error rather than a warning (now we know it should have been important!)

Also, on reflection, since this is on the subscription update, I think it would be better to leave `_state` unchanged rather than setting to `0`. There is a decent chance that the state was set currently (we opportunistically set the internal state).